### ISSUE TITLE
Style the toolbar

### ISSUE THREAD

**Issue by creativetrendsapps** [eliseomartelli/SimpleCustomTabs#4]
Hi, is there away to set the toolbar background to match my apps theme? In Folio I have 41 different themes, lol. But I'd like to have the toolbar be set to the current theme and not just a set color. Is there any option to do this?

**Comment by eliseomartelli** [comment]
Hi!
I've just released 1.4.1 to address this.
Take a look at this:

``` java
TypedValue typedValue = new TypedValue();
Theme theme = context.getTheme();
theme.resolveAttribute(R.attr.theme_color, typedValue, true);
int color = typedValue.data;
style.setToolbarColorInt(color);
```

From: http://stackoverflow.com/questions/17277618/get-color-value-programmatically-when-its-a-reference-theme

**Comment by creativetrendsapps** [comment]
Perfect! I'll try this in my next release.
